src/dags/wior.py

```diff
@@ -0,0 +1,248 @@
+import json
+import logging
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.postgres_operator import PostgresOperator
+
+from common.db import DatabaseEngine
+from environs import Env
+
+from ogr2ogr_operator import Ogr2OgrOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from pgcomparator_cdc_operator import PgComparatorCDCOperator
+from sqlalchemy_create_object_operator import SqlAlchemyCreateObjectOperator
+
+
+from common import (
+    default_args,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    SHARED_DIR,
+    MessageOperator,
+)
+
+from postgres_check_operator import (
+    PostgresMultiCheckOperator,
+    COUNT_CHECK,
+    GEO_CHECK,
+)
+
+from sql.wior import (
+    DROP_COLS,
+    SQL_DROP_TMP_TABLE,
+    SQL_GEOM_CONVERT,
+    SQL_ADD_PK,
+    SQL_SET_DATE_DATA_TYPES,
+)
+
+dag_id = "wior"
+variables = Variable.get(dag_id, deserialize_json=True)
+data_endpoint = variables["data_endpoints"]["wfs"]
+tmp_dir = f"{SHARED_DIR}/{dag_id}"
+data_file = f"{tmp_dir}/{dag_id}.geojson"
+env = Env()
+db_conn = DatabaseEngine()
+password = env("AIRFLOW_CONN_WIOR_PASSWD")
+user = env("AIRFLOW_CONN_WIOR_USER")
+base_url = env("AIRFLOW_CONN_WIOR_BASE_URL")
+total_checks = []
+count_checks = []
+geo_checks = []
+
+
+# define logger for output to console
+logger = logging.getLogger(__name__)
+
+
+# needed to put quotes on elements in geotypes for SQL_CHECK_GEO
+def quote(instr: str) -> str:
+    return f"'{instr}'"
+
+
+class DataSourceError(Exception):
+    pass
+
+
+# data connection
+def get_data() -> None:
+    """calling the data endpoint"""
+    # get data
+    data_url = f"{base_url}/{data_endpoint}"
+    data_request = requests.get(data_url, auth=HTTPBasicAuth(user, password))
+    # store data
+    if data_request.status_code == 200:
+        try:
+            data = data_request.json()
+        except json.decoder.JSONDecodeError:
+            logger.error(f"Failed to convert request output to json for url {data_url}")
+            raise
+        with open(f"{data_file}", "w") as file:
+            file.write(json.dumps(data))
+    else:
+        raise DataSourceError(f"HTTP status code: {data_request.status_code}")
+
+
+with DAG(
+    dag_id,
+    default_args=default_args,
+    template_searchpath=["/"],
+    user_defined_filters=dict(quote=quote),
+    description="Werken (projecten) in de openbare ruimte.",
+) as dag:
+
+    # 1. Post info message on slack
+    slack_at_start = MessageOperator(
+        task_id="slack_at_start",
+        http_conn_id="slack",
+        webhook_token=slack_webhook_token,
+        message=f"Starting {dag_id} ({DATAPUNT_ENVIRONMENT})",
+        username="admin",
+    )
+
+    # 2. Create temp directory to store files
+    mkdir = BashOperator(task_id="mkdir", bash_command=f"mkdir -p {tmp_dir}")
```

Comment:
@ngaranko Has created an ENV var vor the temporary dir. (No sure if that is merged already). Better to use that ENV var.

Reply:
It is used here: https://github.com/Amsterdam/dataservices-airflow/pull/197/files#diff-bdad4f2581886fbd705c54ceba3027c7c835a8be667ec71c9951d131e071d50cR47
:heavy_check_mark:

Reply:
Check :+1: